source/references/glossary.markdown

```diff
@@ -1,82 +1,127 @@
 ---
-title: Glossary
+title: Glossary of Puppet Vocabulary
 layout: default
+canonical: "/references/glossary.html"
```

Comment:
Do we ever want to consider versioning the glossary? Or is it a bad fit for the glossary to describe how term has different or new meanings across versions?